testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -1058,3 +1076,66 @@
     Then I should see a "System properties changed" text
   )
 end
+
+# Select first system listed
+When(/^I select the first system in the list$/) do
+  within(:xpath, '//section') do
+    row = find(:xpath, "//div[@class='table-responsive']/table/tbody/tr[.//td]", match: :first)
+    # click first link in first row
+    within(row) do
+      first('a').click
+    end
+  end
+end
```

Comment:
Yeah, good.. very similar to:
```When(/^I check the first row in the list$/) do
  within(:xpath, '//section') do
    row = find(:xpath, "//div[@class='table-responsive']/table/tbody/tr[.//td]", match: :first)
    row.find(:xpath, ".//input[@type='checkbox']", match: :first).set(true)
  end
end
```

I'd like to reduce duplicated code, maybe we can extract the common part to a function in commonlib.rb.
Anyway, just a thought, can be done "someday"